src/components/Registration/Kiosk/ScannerCamera.vue

```diff
@@ -25,7 +27,9 @@ const validQRCode = ref(true)
 const decode = () => {
   // check if QRCodeValue is valid and conforms to what is needed over here
   showPrintModal.value = true
-  console.log(scannerStore.QRCodeValue)
+  const validQR = scannerStore.isValidQRCode(scannerStore.stringModifier(scannerStore.QRCodeValue))
```

Comment:
this section and scanSearch file uses the same code. u can just leverage on store to prevent dup functions